src/styles/footer/footer.css

```diff
@@ -33,12 +33,17 @@
   margin: 1.5rem auto;
 }
 
-.info-block__link-content {
+.info-block__link {
   transition: transform 0.2s;
 }
 
-.info-block__link-content:hover,
-.info-block__link-content:focus {
+.info-block__link svg {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.info-block__link:hover,
+.info-block__link:focus {
   transform: scale(1.1);
```

Comment:
this will increase the whole link element, I guess we should only increase svg size